pkg/logs/input/kubernetes/kubelet.go

```diff
@@ -0,0 +1,101 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2018 Datadog, Inc.
+
+// +build kubelet
+
+package kubernetes
+
+import (
+	"time"
+
+	"github.com/DataDog/datadog-agent/pkg/util/kubernetes/kubelet"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+)
+
+const (
+	// How long should we wait before scanning for new pods.
+	// This can't be too low to avoid putting too much pressure on the kubelet.
+	// Also, currently KubeUtil has an internal cache duration, there is no
+	// point setting the period lower than that.
+	podScanPeriod = 10 * time.Second
```

Comment:
why do we need to scan pods? Don't we get notified of started and stopped pods?

Reply:
I'm surprised this logic is part of the client and not the AD itself

Reply:
as we are not fully plugged to the AD, we need to stand with a polling logic for now, this is very likely to disappear in the near future.

Reply:
Could we add a note here stating that the AD should replace this logic?